docs/jwst/datamodels/fits_files.rst

```diff
@@ -3,7 +3,16 @@ FITS file structures and contents
 
 Here we describe the structure and content of the most frequently used
 forms of FITS files for JWST science data products. Each type of FITS
-file is the result of serialization of a corresponding data model.
+file is the result of serialization of a corresponding data model. All
+JWST pipeline input and output images, with the exception of a few
+reference files, are serialized as FITS files. The asdf representation
```

Comment:
Nitpicky, but "asdf" should be in all caps, i.e. "ASDF", because it's an acronym, like FITS.